pkg/server/endpoints/registration/handler.go

```diff
@@ -45,10 +45,8 @@ func (h *Handler) CreateEntry(
 	ctx context.Context, request *common.RegistrationEntry) (
 	response *registration.RegistrationEntryID, err error) {
 
-	counter, err := h.startCall(ctx, telemetry.RegistrationAPI, telemetry.Entry, telemetry.Create)
-	if err != nil {
-		return nil, err
-	}
+	counter := telemetry.RegistrationAPICreateEntryCall(h.Metrics)
+	addCallerIDLabel(ctx, counter)
```

Comment:
nit: i'd prefer the defer come right after the counter is initialized to drive home the point that it needs to happen